official/resnet/resnet_run_loop.py

```diff
@@ -355,21 +370,35 @@ def resnet_main(flags, model_function, input_function, shape=None):
       allow_soft_placement=True)
 
   # Set up a RunConfig to save checkpoint and set session config.
-  run_config = tf.estimator.RunConfig().replace(save_checkpoints_secs=1e9,
-                                                session_config=session_config)
+  if not flags.use_distribution_strategy:
```

Comment:
Proposition: We should just use DistStrat in all cases in this model now. Opinions? @guptapriya and @robieta for thoughts.

Reply:
I think that's a good idea. When we originally added these flags, we were not removing the previous multi gpu approach. So we didn't want to enable if by default. But I think now it makes sense. `OneDeviceStrategy` should pretty much do what having no distribution strategy does.